skops/card/_model_card.py

```diff
@@ -372,6 +372,26 @@ def add_metrics(self, **kwargs: str) -> "Card":
             self._eval_results[metric] = value
         return self
 
+    def _load_model(self, model: Any) -> Any:
+        """Loads the model if provided a file path.
+
+        Parameters
+        ----------
+        model : Any
+            Str or model instance.
+
+        Returns
+        -------
+        model : object 
+            Model instance.
+        """
+        if isinstance(model, str):
+            model_path = Path(model)
+            if not model_path.exists():
+                raise ValueError("Model file does not exist")
+            model = skops.io.load(model)
```

Comment:
As of now, we also support pickle files. For this to work, it would be good to add a line that loads a pickle. skops.io.load() loads a model file with skops extension, which is essentially a .zip.